app/src/main/java/org/traccar/client/NetworkManager.java

```diff
@@ -53,16 +53,15 @@ public boolean isOnline() {
     public void start() {
         // Aina 15-Jan-2019, Emergency alert on broadcast
         preferences = PreferenceManager.getDefaultSharedPreferences(context);
-        broadcast = preferences.getString(MainFragment.KEY_BROADCAST, context.getString(R.string.settings_broadcast));
+        broadcast = preferences.getString(MainFragment.KEY_BROADCAST, " ");
```

Comment:
Why is there a space here?

Reply:
Just to ensure that the string isn't completely empty even if there is no KEY_BROADCAST written in preferences.